README.md

```diff
@@ -23,10 +25,61 @@ ember install ember-frost-object-details
 ```
 
 ## API
-Coming soon
+|Component| Attribute | Type | Value | Description |
+|---------| --------- | ---- | ----- | ----------- |
+| `{{frost-object-details}}` | `defaultRoute` | `string` | | Default entry route for object details view (Mandatory) |
+| `{{routes.view}}` | `detailsRouteIndex` | `number` | |  Provide left/right slide animation when transition between two view routes based on comparing their detailsRouteIndex value (Optional) |
```

Comment:
is this left-to-right animation or choice of left Or right animation? Please clarify in the readme